### ISSUE TITLE
[Bug]: v1.4.1 TypeError Cannot read properties of null (reading 'map')

### ISSUE THREAD

**Issue by kjy5** [mgmeyers/obsidian-kanban#648]
### Describe the bug

Unable to startup Kanban view with v1.4.1 update due to a TypeError for reading map used in new date colors

### Expected Behavior

Startup properly and view Kanban boards

### Steps to reproduce

1. Install/upgrade to v1.4.1
2. Open existing Kanban or create a new one
3. See nothing and open dev console to see error

### Relevant errors (if available)

```shell
Uncaught (in promise) TypeError: Cannot read properties of null (reading 'map')
    at wo (plugin:obsidian-kanban:23:92578)
    at eval (plugin:obsidian-kanban:70:5871)
    at Object.lr [as useMemo] (plugin:obsidian-kanban:23:78401)
    at ht.ew [as constructor] (plugin:obsidian-kanban:70:5776)
    at ht.ax [as render] (plugin:obsidian-kanban:10:8852)
    at Ml (plugin:obsidian-kanban:10:6533)
    at Hf (plugin:obsidian-kanban:10:2768)
    at Ml (plugin:obsidian-kanban:10:6803)
    at Hf (plugin:obsidian-kanban:10:2768)
    at Ml (plugin:obsidian-kanban:10:6803)
```

### Operating system

macOS

**Comment by mgmeyers** [comment]
@kjy5 Yeah, apologies for that, just released 1.4.2 that fixes this.

**Comment by kjy5** [comment]
Just saw that too! Thanks for the quick turn-around!
